config/schema/synonyms.yml

```
@@ -102,6 +102,7 @@
 # Unofficial or deprecated terms should expand to the preferred term
 - search: ancestral visa => ancestry visa
 - search: bedroom tax => bedroom tax, spare bedroom, spare room
+- search: brexit => brexit, eu exit
```

This is also configured in such a way that "EU Exit" is the preferred term for "Brexit". Is this the case given https://www.gov.uk/government/brexit ?
```suggestion
- search: eu exit => eu exit, brexit
```

---

@andrewgarner, I'm not 100% following, what do you mean by the preferred term?

Looking at the [docs](https://github.com/alphagov/rummager/blob/master/doc/schemas.md#synonyms) I suppose this might be worth changing to `brexit, eu exit => brexit, eu exit`, which would lead to "a search for brexit or eu exit will return documents with either term used". 

At the moment this change will mean brexit will respond with documents that reference the brexit or eu exit terms.

Is there some other logic that would impact this?

---

This block of configuration has the comment "Unofficial or deprecated terms should expand to the preferred term". My understanding looking at how other synonyms have been entered is that the unofficial or deprecated term is entered first.

So `search: brexit => brexit, eu exit` would suggest that `brexit` is the deprecated term, but any search for `brexit` should also include matches for `eu exit`.

You're right though, `brexit, eu exit => brexit, eu exit` might be what is needed since neither of these terms are used consistently. Or maybe it should be `eu exit, brexit => eu exit, brexit` ?!

---

Yeah sorry this is a bit confusing. I put this synonym under this comment as Brexit is an unofficial term (like `xmas => xmas, christmas`). 

I changed it and moved it under the 'same meanings' comment.

---

👍